.github/workflows/nightly-checks.yml

```diff
@@ -5,21 +5,56 @@ on:
     - cron: '0 7 * * *'
 
 jobs:
-  build:
-    name: Build packages and upload artifacts
```

Comment:
❓ Is the build handled somewhere else now?

Reply:
Yup, the build process is handled in line 37 within the `publish-cmd`.